lesson-4/JAMROZ-week 4 homework/3d art/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <title>3D Art</title>
+  </head>
+  <body>
+
+
+    <h1>3D Art</h1>
+
+    <img src="images/wood-sculpture.jpg" width="300px" />
```

Comment:
You'll need to make this URL `../images/wood-sculpture.jpg` for this image to show up since you're inside of the `3d art` folder. Also just a reminder to name your folders with all lowercase and hyphens instead of spaces.